src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -190,11 +190,10 @@ public void shouldNotSendNotificationForAnAdjournedRequestForAPaperHearing() thr
     }
 
     @Test
-    @Parameters(
-            {
-                    "oral, 1afd89f9-9935-4acb-b4f6-ba708b03a0d3, 4bba0b5d-a3f3-4fd9-a845-26af5eda042e",
-                    "paper, a64bce9a-9162-47ca-b3e7-cf5f85ca7bdc, f5b61f94-0b2b-4e8e-9c25-56e9830df7d4"
-            })
+    @Parameters({
```

Comment:
@PaulVM , this is very annoying. You haven't changed any code here, only the formatting, which I presume is because you use Visualcode. You should either make sure VisualCode follows the same java standard formatting or move onto IntelliJ.

Reply:
Nope I changed it by hand because ({   }) looks nicer than
(
{
)}
to my eyes at least. I guess it's a personal taste thing. Do we have a style document I can follow in future?